.gitignore

```diff
@@ -1,12 +1,10 @@
 # Whitelist dotfiles
 .*
-!deps/**/.*
-!test/fixtures/**/.*
-!tools/eslint/**/.*
-!tools/doc/node_modules/**/.*
+!deps/**
```

Comment:
I'm guessing this would trump https://github.com/nodejs/node/pull/17363

Reply:
I'll check. exclusion rules seem to be tricky (CSS style), but I think these are stronger:
https://github.com/nodejs/node/blob/a84f58d28f705e30cd07dff890a9fe82389fa986/.gitignore#L111-L120

Reply:
should it at the very least be at the bottom to avoid other rules combatting it?

Reply:
What I saw while testing is that most of the rules below apply for `/deps` (only conflict is debug/release), so IMO it makes sense to keep it up here.